### ISSUE TITLE
I can't text my gpt.

### ISSUE THREAD

**Issue by KenanExe** [bayeggex/Arduino-AI-Chat-Library#2]
![image](https://github.com/bayeggex/Arduino-AI-Chat-Library/assets/99500843/09e0436f-64f5-40e6-8e02-48fb0fd8b81e)
Dude u don't know json data?!
rly?

**Comment by bayeggex** [comment]
oh, thats a small bug in Library. If you use esp8266 this is a issue just change the cpp file or wait for the new update

```
String AIChatbot::makeHttpRequest(const String& url, const String& payload, const String& apiKey) {
    HTTPClient http;
    WiFiClient client;

    http.begin(client, url);
    http.addHeader("Content-Type", "application/json");
    http.addHeader("Authorization", "Bearer " + apiKey);

    int httpResponseCode = http.POST(payload);
    String response;

    if (httpResponseCode > 0) {
        response = http.getString();
    } else {
        response = "Error on sending POST: " + String(httpResponseCode);
    }

    http.end();
    return response;
     }
```

**Comment by bayeggex** [comment]
It Fixed you can use it in new version V1.2.0
